src/CAREUI/misc/PrintPreview.tsx

```diff
@@ -20,7 +20,13 @@ export default function PrintPreview(props: Props) {
 
   return (
     <Page title={props.title}>
-      <div className="mx-auto my-8 w-[50rem]">
+      {/* INCREASED WIDTH TO REMOVE SCROLLBAR  */}
+
+      {/* PREVIOUS WIDTH = 50REM
+
+      UPDATED WIDTH = 64REM */}
+
+      <div className="mx-auto my-8 lg:w-[64rem]">
```

Comment:
This is a reusable component meant to be used to show how to print would look like. Instead of modifying this component, let's have two pages: 

1. The one that shows the Investigation Details with all the interactive components such as update details, select menu etc...

2. And a print button that takes you to the print preview page, which will not have any interactive components in the print preview. Select Menu doesn't and other interactive elements should not be part of the "print preview".

Reply:
sure